nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/QueryDatabaseTable.java

```diff
@@ -127,6 +129,7 @@
             .defaultValue("0")
             .required(true)
             .addValidator(StandardValidators.NON_NEGATIVE_INTEGER_VALIDATOR)
+            .expressionLanguageSupported(true)
             .build();
 
     public QueryDatabaseTable() {
```

Comment:
I agree that Initial Max Value would be useful for GenerateTableFetch as well. https://issues.apache.org/jira/browse/NIFI-2583 was written and implemented by a community member, his focus was on QueryDatabaseTable at the time. Jira is down ATM but I will write an improvement Jira for GenerateTableFetch and post the link here.